### PR TITLE
Changes required to work with the latest version of node 2.0

### PR DESCRIPTION
**Pull request by rafal-ch**

### CODE REVIEW

File: lib/cli/parse.rs [hunk 1]
```diff
@@ -782,30 +781,37 @@ pub(super) fn entity_identifier(entity_identifier: &str) -> Result<EntityIdentif
...
-    maybe_paid_amount: Option<U512>,
+    maybe_paid_amount: Option<u64>,
 ) -> Result<PricingMode, CliError> {
     match pricing_mode_identifier_str.to_lowercase().as_str() {
         "classic" => {
-            if maybe_gas_price_str.is_empty() {
+            if maybe_gas_price_tolerance_str.is_empty() {
                 return Err(CliError::InvalidArgument {
                     context: "gas-price",
                     error: "Gas price is required".to_string(),
```
Comment (alsrdn) [inline lib/cli/parse.rs]: nit: maybe we should change the message here too: `Gas price tolerance required`

File: src/transaction/creation_common.rs [hunk 2]
```diff
@@ -355,8 +358,57 @@ pub(super) mod paid_amount {
...
+    }
+}
+
+pub(super) mod standard_payment {
+    use super::*;
+    pub(in crate::transaction) const ARG_NAME: &str = "standard-payment";
+
+    const ARG_VALUE_NAME: &str = common::ARG_STRING;
+
+    const ARG_HELP: &str = "Standard payment.";
```
Comment (alsrdn) [inline src/transaction/creation_common.rs]: nit: a more helpful help string would be nice here. Something like: `Flag to determine if this transaction uses standard or custom payment`.

**Review by alsrdn — APPROVED** [review]

**Review by jacek-casper — APPROVED** [review]

File: lib/cli/parse.rs (hunk 1 above)
Comment (rafal-ch) [reply lib/cli/parse.rs]: Updated in 44a0e28c611d6623c8a6ac2ccafa3f4d27cc5131

File: src/transaction/creation_common.rs (hunk 2 above)
Comment (rafal-ch) [reply src/transaction/creation_common.rs]: Updated in 041470958d8151e2cb405baffe724234ddc591b9

**Pull request merged**